runtime/j9vm/java11vmi.c

```diff
@@ -423,6 +425,7 @@ addModuleDefinition(J9VMThread * currentThread, J9Module * fromModule, const cha
 		retval = addMulPackageDefinitions(currentThread, fromModule, packages, numPackages);
 		if (ERRCODE_SUCCESS == retval) {
 			BOOLEAN const success = (0 == hashTableAtPut(classLoader->moduleHashTable, (void*)&fromModule, TRUE));
+			Trc_MODULE_invokeHashTableAtPut(currentThread, "addModuleDefinition", classLoader->moduleHashTable, &fromModule, fromModule, "true");
```

Comment:
Similar question here - do we want to log the moduleHashTable or the ClassLoader pointer?

Reply:
Yeah, same thing here.